projects/control-service/projects/pipelines_control_service/src/main/resources/application.properties

```diff
@@ -69,15 +69,15 @@ datajobs.deployment.configuration.synchronization.task.enabled=${DATAJOBS_DEPLOY
 datajobs.deployment.configuration.synchronization.task.interval.ms=${DATAJOBS_DEPLOYMENT_CONFIGURATION_SYNCHRONIZATION_TASK_INTERVAL_MS:60000}
 # The data job deployments' synchronization task initial delay is the period (expressed in milliseconds) between control service
 # start and the first time a synchronization process is started by the control service instance. The time unit is milliseconds.
-datajobs.deployment.configuration.synchronization.task.initial.delay.ms=${DATAJOBS_DEPLOYMENT_CONFIGURATION_SYNCHRONIZATION_TASK_INTERVAL_DELAY_MS:10000}
+datajobs.deployment.configuration.synchronization.task.initial.delay.ms=${DATAJOBS_DEPLOYMENT_CONFIGURATION_SYNCHRONIZATION_TASK_INITIAL_DELAY_MS:10000}
 # The core pool size of the ThreadPoolExecutor for synchronizing data job deployments.
-datajobs.deployment.configuration.synchronization.task.corePoolSize=5
+datajobs.deployment.configuration.synchronization.task.corePoolSize=${DATAJOBS_DEPLOYMENT_CONFIGURATION_SYNCHRONIZATION_TASK_CORE_POOL_SIZE:5}
 # The maximum pool size of the ThreadPoolExecutor for synchronizing data job deployments.
-datajobs.deployment.configuration.synchronization.task.maxPoolSize=10
+datajobs.deployment.configuration.synchronization.task.maxPoolSize=${DATAJOBS_DEPLOYMENT_CONFIGURATION_SYNCHRONIZATION_TASK_MAX_POOL_SIZE:5}
 # The queue capacity of the ThreadPoolExecutor for synchronizing data job deployments.
-datajobs.deployment.configuration.synchronization.task.queueCapacity=2000
+datajobs.deployment.configuration.synchronization.task.queueCapacity=${DATAJOBS_DEPLOYMENT_CONFIGURATION_SYNCHRONIZATION_TASK_QUEUE_CAPACITY:4000}
```

Comment:
is there a reason for doubling the default here?

Reply:
Yes, the data jobs in Supercollider are more than 2000 and this is a problem for us.